distro/otel_test.go

```diff
@@ -143,6 +143,9 @@ func TestMain(m *testing.M) {
 }
 
 func TestRunJaegerExporter(t *testing.T) {
+	// Do not use the default metrics exporter.
+	t.Setenv("OTEL_METRICS_EXPORTER", "none")
```

Comment:
This is needed as otherwise the `sdk.Shutdown` fails as it cannot export the runtime metrics.

I am considering adding the following (pseudo-)code 

```go
	cleanup := setenv("OTEL_TRACES_EXPORTER", "none") 
	defer cleanup()
	cleanup = setenv("OTEL_METRICS_EXPORTER", "none") 
	defer cleanup()
```

Then the tests would explicitly e.g. `t.Setenv("OTEL_TRACES_EXPORTER", "")` when a default exporter is needed in test.

Something like this https://github.com/signalfx/splunk-otel-go/blob/64f4431794100cd5230e51535705865df007e2bf/distro/helpers_test.go would be brought back 😉

 I would rather do this in a separate PR to make reviewing easier.

Reply:
https://github.com/signalfx/splunk-otel-go/issues/2260